src/html/modules/common/head.html

```diff
@@ -7,6 +7,9 @@
 <link rel="icon" href="favicon.ico" type="image/x-icon">
 <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600|Source+Sans+Pro:400,600,700|PT+Sans"
     rel="stylesheet">
+<link href='https://api.mapbox.com/mapbox-gl-js/v2.0.0/mapbox-gl.css' rel='stylesheet' />
+<script src='https://api.mapbox.com/mapbox-gl-js/v2.0.0/mapbox-gl.js'></script>
```

Comment:
`common/head.html` file is for content shared between all pages.
mapbox files should be appended only to a page where the map will be.